velox/docs/functions/coverage.rst

```diff
@@ -135,10 +135,10 @@ Here is a list of all scalar and aggregate Presto functions with functions that
     any_match                                 date_add                                  is_json_scalar                            repeat                                    st_relate                                     approx_set
     array_average                             date_diff                                 is_nan                                    :func:`replace`                           st_startpoint                                 :func:`arbitrary`
     :func:`array_distinct`                    date_format                               is_subnet_of                              :func:`reverse`                           st_symdifference                              :func:`array_agg`
-    array_dupes                               date_parse                                jaccard_index                             rgb                                       st_touches                                    :func:`avg`
+    :func:`array_duplicates`                  date_parse                                jaccard_index                             rgb                                       st_touches                                    :func:`avg`
```

Comment:
Thank you for noticing that coverage maps need updating as well. We update these in bulk using the procedure described in https://github.com/facebookincubator/velox/tree/main/velox/functions/prestosql/coverage#readme and submit changes as separate PRs, e.g. #454 . Hence, let's not update these here.